### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by donnyh13**
### Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.<br>
Please ensure you have read and noticed the checklist below.

### Checklist

Put an `x` in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We are here to help!<br>
This is simply a reminder of what we are going to look for before merging your code.

- [x] I have read and noticed the [CODE OF CONDUCT](https://github.com/mlipok/Au3LibreOffice/tree/main/Docs/CODE_OF_CONDUCT.md) document
- [x] I have read and noticed the [CONTRIBUTING](https://github.com/mlipok/Au3LibreOffice/tree/main/Docs/CONTRIBUTING.md) document
- [x] I have added necessary documentation or screenshots (if appropriate)
- [ ] I have tested my changes or additions for proper functionality.
- [x] I have followed, as best as possible, the same function layout and coding style of other existing functions/code.

### Types of changes

Please check `x` the type of change your PR introduces:

- [ ] Bugfix (change which fixes an issue)
- [ ] Feature (change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (functional, structural)
- [x] Documentation content changes
- [ ] Other (please describe) <br>
**	|-->:

### What is the current behavior?

Please describe the current behavior that you are modifying, or link to a relevant issue. <br>
**	|-->:

### What is the new behavior?

Please describe the behavior or changes that are being added by this PR. <br>
**	|-->:

### Influences and relationship to other functionality

Describe how the changes will affect other functions, potential script breaking changes, etc. <br>
**	|-->:

### Additional context

Add any other context about the problem here. <br>
**	|-->: Modify ReadMe to add detail. Modify au3 File header to describe what each does. Remove repetitive wording. 

### System under test

Please complete the following information.

- OS: [e.g. Windows 10]
- OS Arch.: [e.g. X64]
- LibreOffice Component [e.g. Main, Writer, Calc, ]
- LibreOffice version [e.g. 7.3.4.2]

### CODE REVIEW

File: LibreOfficeWriter_Font.au3 [hunk 1]
```diff
@@ -6,23 +6,12 @@
 #include "LibreOfficeWriter_Internal.au3"
 
 ; #INDEX# =======================================================================================================================
-; Title .........: Libre Office Writer (LOWriter)
+; Title .........: LibreOffice UDF
 ; AutoIt Version : v3.3.16.1
-; UDF Version    : 0.0.0.3
-; Description ...: Provides basic functionality through Autoit for interacting with Libre Office Writer.
+; Description ...: Provides basic functionality through AutoIt for Listing and querying available L.O. Writer Font Styles.
```
Comment (mlipok) [inline LibreOfficeWriter_Font.au3]: Font Styles ?

File: LibreOfficeWriter_Field.au3 [hunk 2]
```diff
@@ -8,23 +8,12 @@
 #include "LibreOfficeWriter_Doc.au3"
 
 ; #INDEX# =======================================================================================================================
-; Title .........: Libre Office Writer (LOWriter)
+; Title .........: LibreOffice UDF
 ; AutoIt Version : v3.3.16.1
-; UDF Version    : 0.0.0.3
-; Description ...: Provides basic functionality through Autoit for interacting with Libre Office Writer.
+; Description ...: Provides basic functionality through AutoIt for Inserting and Modifying L.O. Writer Fields.
```
Comment (mlipok) [inline LibreOfficeWriter_Field.au3:13]: Fields ?

File: LibreOfficeWriter_Doc.au3 [hunk 3]
```diff
@@ -8,23 +8,12 @@
 #include "LibreOfficeWriter_Par.au3"
 
 ; #INDEX# =======================================================================================================================
-; Title .........: Libre Office Writer (LOWriter)
+; Title .........: LibreOffice UDF
 ; AutoIt Version : v3.3.16.1
-; UDF Version    : 0.0.0.3
-; Description ...: Provides basic functionality through Autoit for interacting with Libre Office Writer.
+; Description ...: Provides basic functionality through AutoIt for Creating, Modifying, Closing, Saving, Searching, etc. L.O. Writer documents.
```
Comment (mlipok) [inline LibreOfficeWriter_Doc.au3:13]: documents ?

File: LibreOfficeWriter_DirectFormating.au3 [hunk 4]
```diff
@@ -10,23 +10,12 @@
 #include "LibreOfficeWriter_Page.au3"
 
 ; #INDEX# =======================================================================================================================
-; Title .........: Libre Office Writer (LOWriter)
+; Title .........: LibreOffice UDF
 ; AutoIt Version : v3.3.16.1
-; UDF Version    : 0.0.0.3
-; Description ...: Provides basic functionality through Autoit for interacting with Libre Office Writer.
+; Description ...: Provides basic functionality through AutoIt for Modifying and Applying Direct Character and Paragraph formating in L.O. Writer.
```
Comment (mlipok) [inline LibreOfficeWriter_DirectFormating.au3:15]: Paragraph ?

File: LibreOfficeWriter_Char.au3 [hunk 5]
```diff
@@ -8,23 +8,12 @@
 #include "LibreOfficeWriter_Font.au3"
 
 ; #INDEX# =======================================================================================================================
-; Title .........: Libre Office Writer (LOWriter)
+; Title .........: LibreOffice UDF
 ; AutoIt Version : v3.3.16.1
-; UDF Version    : 0.0.0.3
-; Description ...: Provides basic functionality through Autoit for interacting with Libre Office Writer.
+; Description ...: Provides basic functionality through AutoIt for Creating, Modifying, and Applying L.O. Writer Character Styles.
```
Comment (mlipok) [inline LibreOfficeWriter_Char.au3:13]: styles ?

File: LibreOfficeWriter_FootEndNotes.au3 [hunk 6]
```diff
@@ -6,23 +6,12 @@
 #include "LibreOfficeWriter_Internal.au3"
 
 ; #INDEX# =======================================================================================================================
-; Title .........: Libre Office Writer (LOWriter)
+; Title .........: LibreOffice UDF
 ; AutoIt Version : v3.3.16.1
-; UDF Version    : 0.0.0.3
-; Description ...: Provides basic functionality through Autoit for interacting with Libre Office Writer.
+; Description ...: Provides basic functionality through AutoIt for Creating, Modifying, and Inserting L.O. Writer Footnotes and Endnotes.
```
Comment (mlipok) [inline LibreOfficeWriter_FootEndNotes.au3]: Foot/End notes ?

File: LibreOfficeWriter_Frame.au3 [hunk 7]
```diff
@@ -8,23 +8,12 @@
 #include "LibreOfficeWriter_Doc.au3"
 
 ; #INDEX# =======================================================================================================================
-; Title .........: Libre Office Writer (LOWriter)
+; Title .........: LibreOffice UDF
 ; AutoIt Version : v3.3.16.1
-; UDF Version    : 0.0.0.3
-; Description ...: Provides basic functionality through Autoit for interacting with Libre Office Writer.
+; Description ...: Provides basic functionality through AutoIt for Creating, Modifying, and Applying L.O. Writer Frame Styles. Also for Creating, and Modifying Frames.
```
Comment (mlipok) [inline LibreOfficeWriter_Frame.au3:13]: Frames ?

File: LibreOfficeWriter_Helper.au3 [hunk 8]
```diff
@@ -7,23 +7,12 @@
 #include "LibreOfficeWriter_Page.au3"
 
 ; #INDEX# =======================================================================================================================
-; Title .........: Libre Office Writer (LOWriter)
+; Title .........: LibreOffice UDF
 ; AutoIt Version : v3.3.16.1
-; UDF Version    : 0.0.0.3
-; Description ...: Provides basic functionality through Autoit for interacting with Libre Office Writer.
+; Description ...: Functions used for creating, modifying and retrieving data for use in various functions in LibreOffice UDF.
```
Comment (mlipok) [inline LibreOfficeWriter_Helper.au3:12]: should be Page ?

File: LibreOfficeWriter_Images.au3 [hunk 9]
```diff
@@ -9,23 +9,12 @@
 #include "LibreOfficeWriter_Page.au3"
 
 ; #INDEX# =======================================================================================================================
-; Title .........: Libre Office Writer (LOWriter)
+; Title .........: LibreOffice UDF
 ; AutoIt Version : v3.3.16.1
-; UDF Version    : 0.0.0.3
-; Description ...: Provides basic functionality through Autoit for interacting with Libre Office Writer.
+; Description ...: Provides basic functionality through AutoIt for Inserting and Modifying Images in L.O. Writer.
```
Comment (mlipok) [inline LibreOfficeWriter_Images.au3:14]: Images ?

File: LibreOfficeWriter_Internal.au3 [hunk 10]
```diff
@@ -7,23 +7,12 @@
 #include <WinAPIGdiDC.au3>
 
 ; #INDEX# =======================================================================================================================
-; Title .........: Libre Office Writer (LOWriter)
+; Title .........: LibreOffice UDF
 ; AutoIt Version : v3.3.16.1
-; UDF Version    : 0.0.0.3
-; Description ...: Provides basic functionality through Autoit for interacting with Libre Office Writer.
+; Description ...: Various functions for internal data processing, data retrieval, data processing, retrieving and applying settings for LibreOffice UDF.
```
Comment (mlipok) [inline LibreOfficeWriter_Internal.au3]: ok

File: LibreOfficeWriter_Num.au3 [hunk 11]
```diff
@@ -8,23 +8,12 @@
 #include "LibreOfficeWriter_Char.au3"
 
 ; #INDEX# =======================================================================================================================
-; Title .........: Libre Office Writer (LOWriter)
+; Title .........: LibreOffice UDF
 ; AutoIt Version : v3.3.16.1
-; UDF Version    : 0.0.0.3
-; Description ...: Provides basic functionality through Autoit for interacting with Libre Office Writer.
+; Description ...: Provides basic functionality through AutoIt for Creating, Modifying, and Applying L.O. Writer Numbering Styles.
```
Comment (mlipok) [inline LibreOfficeWriter_Num.au3:13]: should be char ?

File: LibreOfficeWriter_Page.au3 [hunk 12]
```diff
@@ -8,23 +8,12 @@
 #include "LibreOfficeWriter_Par.au3"
 
 ; #INDEX# =======================================================================================================================
-; Title .........: Libre Office Writer (LOWriter)
+; Title .........: LibreOffice UDF
 ; AutoIt Version : v3.3.16.1
-; UDF Version    : 0.0.0.3
-; Description ...: Provides basic functionality through Autoit for interacting with Libre Office Writer.
+; Description ...: Provides basic functionality through AutoIt for Creating, Modifying, and Applying L.O. Writer Page Styles.
```
Comment (mlipok) [inline LibreOfficeWriter_Page.au3:13]: Page ? should be Par ?

File: LibreOfficeWriter_Par.au3 [hunk 13]
```diff
@@ -11,23 +11,12 @@
 #include "LibreOfficeWriter_Page.au3"
 
 ; #INDEX# =======================================================================================================================
-; Title .........: Libre Office Writer (LOWriter)
+; Title .........: LibreOffice UDF
 ; AutoIt Version : v3.3.16.1
-; UDF Version    : 0.0.0.3
-; Description ...: Provides basic functionality through Autoit for interacting with Libre Office Writer.
+; Description ...: Provides basic functionality through AutoIt for Creating, Modifying, and Applying L.O. Writer Paragraph Styles.
```
Comment (mlipok) [inline LibreOfficeWriter_Par.au3:16]: Paragraph ? or should be Page ?

File: LibreOfficeWriter_Shapes.au3 [hunk 14]
```diff
@@ -5,23 +5,12 @@
 #include "LibreOfficeWriter_Internal.au3"
 
 ; #INDEX# =======================================================================================================================
-; Title .........: Libre Office Writer (LOWriter)
+; Title .........: LibreOffice UDF
 ; AutoIt Version : v3.3.16.1
-; UDF Version    : 0.0.0.3
-; Description ...: Provides basic functionality through Autoit for interacting with Libre Office Writer.
+; Description ...: Provides basic functionality through AutoIt for Creating, Modifying, and Inserting shapes in L.O. Writer.
```
Comment (mlipok) [inline LibreOfficeWriter_Shapes.au3:10]: Ok.

File: LibreOfficeWriter_Table.au3 [hunk 15]
```diff
@@ -8,23 +8,12 @@
 #include "LibreOfficeWriter_Page.au3"
 
 ; #INDEX# =======================================================================================================================
-; Title .........: Libre Office Writer (LOWriter)
+; Title .........: LibreOffice UDF
 ; AutoIt Version : v3.3.16.1
-; UDF Version    : 0.0.0.3
-; Description ...: Provides basic functionality through Autoit for interacting with Libre Office Writer.
+; Description ...: Provides basic functionality through AutoIt for Creating, Modifying, and Inserting Tables in L.O. Writer.
```
Comment (mlipok) [inline LibreOfficeWriter_Table.au3:13]: Tables ? or Page ?

**Comment by mlipok** [comment]
It appears that in many cases you have confused the description you provided with the file in question.

I hope you can fix it soon.

File: LibreOfficeWriter_Font.au3 (hunk 1 above)
Comment (donnyh13) [reply LibreOfficeWriter_Font.au3]: Sorry, should be "Fonts"

File: LibreOfficeWriter_Field.au3 (hunk 2 above)
Comment (donnyh13) [reply LibreOfficeWriter_Field.au3:13]: Yes, I mean like this:
https://help.libreoffice.org/latest/en-US/text/swriter/guide/fields.html

File: LibreOfficeWriter_Doc.au3 (hunk 3 above)
Comment (donnyh13) [reply LibreOfficeWriter_Doc.au3:13]: You would prefer singular? Or what are your thoughts?

File: LibreOfficeWriter_DirectFormating.au3 (hunk 4 above)
Comment (donnyh13) [reply LibreOfficeWriter_DirectFormating.au3:15]: Yes, some direct formatting is applied specifically for Characters, others, such as Tab Stops,  etc, I believe are applied specifically to paragraphs. At least as best as I can tell.

File: LibreOfficeWriter_Char.au3 (hunk 5 above)
Comment (donnyh13) [reply LibreOfficeWriter_Char.au3:13]: Yes, Libre Office has two ways of applying such things as "Bold", Direct formatting and Character Styles.
https://help.libreoffice.org/7.2/en-US/text/swriter/01/05130002.html

File: LibreOfficeWriter_FootEndNotes.au3 (hunk 6 above)
Comment (donnyh13) [reply LibreOfficeWriter_FootEndNotes.au3]: Okay

File: LibreOfficeWriter_Helper.au3 (hunk 8 above)
Comment (donnyh13) [reply LibreOfficeWriter_Helper.au3:12]: What do you mean?

**Comment by donnyh13** [comment]
> It appears that in many cases you have confused the description you provided with the file in question.
> 
> I hope you can fix it soon.

Sorry just saw this message, have been replying to your individual comments.
 I'll review, again. 
Edit*
Unless there is some strange glitch between my computer and yours, most of the changes are looking correct on my end????

For example the file LibreOfficeWriter_Page.au3 shows these lines:
Removed ~~; Description ...: Provides basic functionality through Autoit for interacting with Libre Office Writer.~~
Added **; Description ...: Provides basic functionality through AutoIt for Creating, Modifying, and Applying L.O. Writer Page Styles.**

And your comment is:
> Page ? should be Par ?

File: LibreOfficeWriter_Frame.au3 (hunk 7 above)
Comment (donnyh13) [reply LibreOfficeWriter_Frame.au3:13]: Yes this file deals with both Frames and Frame Styles.

File: LibreOfficeWriter_Char.au3 (hunk 5 above)
Comment (mlipok) [reply LibreOfficeWriter_Char.au3:13]: but here we have LibreOfficeWriter_Char.au3 not styles.... so I was confused.
But now I get it what you mean.
`Characters Styles` not `Styles`

File: LibreOfficeWriter_DirectFormating.au3 (hunk 4 above)
Comment (mlipok) [reply LibreOfficeWriter_DirectFormating.au3:15]: I was confused but now understand.
Thanks.

File: LibreOfficeWriter_Doc.au3 (hunk 3 above)
Comment (mlipok) [reply LibreOfficeWriter_Doc.au3:13]: Here was my mistake.
Sorry.

File: LibreOfficeWriter_Field.au3 (hunk 2 above)
Comment (mlipok) [reply LibreOfficeWriter_Field.au3:13]: My mistake. I was looking wrong place.
image

File: LibreOfficeWriter_Frame.au3 (hunk 7 above)
Comment (mlipok) [reply LibreOfficeWriter_Frame.au3:13]: my mistake. Sorry.

File: LibreOfficeWriter_Helper.au3 (hunk 8 above)
Comment (mlipok) [reply LibreOfficeWriter_Helper.au3:12]: My mistake.
Sorry.

File: LibreOfficeWriter_Images.au3 (hunk 9 above)
Comment (mlipok) [reply LibreOfficeWriter_Images.au3:14]: My mistake sorry.

File: LibreOfficeWriter_Num.au3 (hunk 11 above)
Comment (mlipok) [reply LibreOfficeWriter_Num.au3:13]: My mistake. Sorry.

File: LibreOfficeWriter_Page.au3 (hunk 12 above)
Comment (mlipok) [reply LibreOfficeWriter_Page.au3:13]: My mistake. sorry.

File: LibreOfficeWriter_Par.au3 (hunk 13 above)
Comment (mlipok) [reply LibreOfficeWriter_Par.au3:16]: My mistake. Sorry.

File: LibreOfficeWriter_Table.au3 (hunk 15 above)
Comment (mlipok) [reply LibreOfficeWriter_Table.au3:13]: My mistake.

File: LibreOfficeWriter_Font.au3 (hunk 1 above)
Comment (mlipok) [reply LibreOfficeWriter_Font.au3]: no.
My mistake. Sorry

**Comment by mlipok** [comment]
I was looking in wrong place:
image

My mistake. Sorry for confusion.

File: LibreOfficeWriter_Field.au3 (hunk 2 above)
Comment (donnyh13) [reply LibreOfficeWriter_Field.au3:13]: Oh Okay. I understand now.

**Comment by donnyh13** [comment]
No problem. Thanks for clarifying.

**Pull request merged**